src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -61,7 +61,7 @@ public ReadOnlyPerson getPerson() {
     public CommandResult execute() {
         try {
             addressBook.addPerson(toAdd);
-            return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
+            return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd, toAdd.getSequenceNumber()));
```

Comment:
If you want to use the sequenceNumber is ReadOnlyPerson then you need to update the interface as well.